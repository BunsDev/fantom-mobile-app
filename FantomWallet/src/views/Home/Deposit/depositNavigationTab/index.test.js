import React from 'react';
import { shallow } from 'enzyme';

// eslint-disable-next-line import/first
import DepositNavigationTab from './index';

test('renders correctly', () => {
  const wrapper = shallow(
    <DepositNavigationTab
      activeTabIndex={3}
      index={5}
      activeTabColor="red"
      tabIfo={{
        tabRenderInfo: 'text',
      }}
    />,
  );
  expect(wrapper).toMatchSnapshot();
});
