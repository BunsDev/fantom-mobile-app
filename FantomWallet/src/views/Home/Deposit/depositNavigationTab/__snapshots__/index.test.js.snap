// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderBottomColor": undefined,
      "borderBottomWidth": 3,
      "flex": 1,
      "height": 44,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 16,
        "fontWeight": "normal",
      }
    }
  >
    text
  </Text>
</TouchableOpacity>
`;
